.github/workflows/ci.yml

```diff
@@ -86,6 +86,26 @@ jobs:
       - name: Run prettier linter
         run: npm run checkformat
 
+  tests-jest:
```

Comment:
this key needs to be added to the `needs` array in the `deploy` job :) don't want to deploy if there are any test failures, jest or storybook